src/mixins/events-delegation.coffee

```diff
@@ -26,7 +26,7 @@ class EventsDelegation extends Mixin
     eachPair events, (event, callback) =>
       unless eventsForObject[event]?
         eventsForObject[event] = {}
-        disposablesForObject[event] = @createEventListener(object, event)
+        disposablesForObject[event] = @createEventListener(object, event, options)
```

Comment:
We could also enable users to define options on an event by event basis but I guess this API is nicer for most of the use cases.